scripts/ktlint-pre-commit

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+echo "********************************"
+echo "Checking code formatting"
+echo "********************************"
+
+ktlint -F --android app/src/**/*.kt data/src/**/*.kt domain/src/**/*.kt testing/src/**/*.kt utility/src/**/*.kt
```

Comment:
Lets remove the -F flag here

Reply:
Done, resolving on behalf of Vinita as it's a small fix.